devops/openshift/database/patroni-deploy.yml

```diff
@@ -182,6 +182,17 @@ objects:
         resources:
           requests:
             storage: ${PVC_SIZE}
+- apiVersion: policy/v1
```

Comment:
Do we have a source of recommendation for Patroni PDB configuration?

Reply:
https://github.com/bcgov/SIMS/pull/2237#discussion_r1313588539

Reply:
https://github.com/bcgov/SIMS/pull/2237#discussion_r1313591274